docs/design/autoware-concepts/difference-from-ai-and-auto.md

```diff
@@ -0,0 +1,22 @@
+# How is Autoware Core/Universe Different from Autoware.AI and Autoware.Auto
```

Comment:
Autoware docs use sentence case for titles [as recommended by the Google developer documentation style guide](https://developers.google.com/style/headings?hl=en)

```suggestion
# How is Autoware Core/Universe different from Autoware.AI and Autoware.Auto
```